add_rental.rb

```diff
@@ -0,0 +1,32 @@
+require './rental'
+
+module CreateRental
+  def add_rental
+    puts "Select a book from the following list by number:\n"
+
```

Comment:
When I select option 3 to create a rental I get an error.

![image](https://user-images.githubusercontent.com/38088818/174105911-451bdfd7-6d7f-4585-afe7-f51632ef62a8.png)

- Kindly look into this and make sure that creating a rental works properly.